clang/lib/CConv/DeclRewriter.cpp

```diff
@@ -112,9 +112,10 @@ void DeclRewriter::rewriteDecls(ASTContext &Context, ProgramInfo &Info,
   // rewrite things like int x, *y, **z;
   GlobalVariableGroups GVG(R.getSourceMgr());
   FieldFinder FF(GVG);
-  for (const auto &D : TUD->decls())
-    GVG.addGlobalDecl(dyn_cast<VarDecl>(D)),
-      FF.TraverseDecl(D);
+  for (const auto &D : TUD->decls()) {
+    GVG.addGlobalDecl(dyn_cast<VarDecl>(D));
+    FF.TraverseDecl(D);
```

Comment:
I asked this in the last revision: What is this doing? "TraverseDecl" is not saying very much. Can you make the name more self-explanatory?

Reply:
I can't rename as that's the function setup by the ASTVisitor Interface. Hid it behind a static function and added a comment explaining.